cadc-access-control/src/main/java/ca/nrc/cadc/ac/client/UserClient.java

```diff
@@ -149,7 +147,7 @@ public void augmentSubject(Subject subject) throws MalformedURLException
 
 	        // augment subject calls are always https with client certs
             URL usersURL = getRegistryClient()
-                .getServiceURL(this.serviceID, Standards.UMS_USERS_01, AuthMethod.CERT);
+                .getServiceURL(this.serviceID, Standards.UMS_USERS_01, getAuthMethod());
```

Comment:
There is no cookie support?  The whole intent of this method is to be used from the UI.